mycli/main.py

```diff
@@ -111,8 +112,11 @@ def __init__(self, sqlexecute=None, prompt=None,
             self.cnf_files = [defaults_file]
 
         # Load config.
-        config_files = ([self.default_config_file] + self.system_config_files +
-                        [self.user_config_file])
+        config_files = ([self.default_config_file] + self.system_config_files)
```

Comment:
Ah, great point! I totally forgot about that.

It looks like pgcli uses Click to provide the default value -- like you suggested.